lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+    }
+   },
+   "source": [
+    "Hence, the classical $\\chi^2$ test looks at whether $\\hat{\\chi}^2$ is consistent with this distribution. If not, it's unlikely that our data came from the assumed model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "In this case, the value of $\\hat{\\chi}^2\\approx104$ doesn't look good in light of the expectation.\n",
+    "<img src=\"graphics/modelcheck-chisq.png\" width=50%>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The probability $P(\\chi^2\\geq\\hat{\\chi}^2|\\nu)$ ($\\sim10^{-10}$ in this case) is called the **$p$-value** or **significance**.\n",
+    "\n",
+    "* If the \"null hypothesis\" (our assumed model, with fitted parameters $[\\hat{a},\\hat{b}]$) is true, we expect the fraction of hypothetical new datasets to have $\\chi^2$ values greater than $\\hat{\\chi}^2$ to be $p$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "The $p$-value is _not_ the probability of the model$(a,b)$ being true. Like the sampling distribution from which it is derived, it characterizes the probability of getting the data given the assumed model and its estimated parameters.\n",
+    "\n",
+    "The result of a classical hypothesis test is of the following form:\n",
+    "\n",
+    "_\"We reject the null hypothesis at the $p$ significance level\"_\n",
+    "\n",
+    "(i.e. on the grounds that it inadequately predicts the data.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Practical Chi-squared Testing\n",
+    "\n",
+    "* We can compute the p-value assuming a chi-squared distribution using `scipy.stats`:\n",
+    "```python\n",
+    "import scipy.stats\n",
+    "chisq = scipy.stats.chi2(Ndof)\n",
+    "pvalue = chisq.sf(chisq_min)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "* The \"reduced chi-squared\", $\\hat{\\chi}^2 = \\chi^2_{\\rm min} / N_{\\rm dof}$, is often used by astronomers to quantify goodness of fit - but note that you need to know the number of degrees of freedom separately from $\\chi^2_{\\rm min}$ to be able to interpret it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "* A useful, quick way to make sense of  $\\chi^2_{\\rm min}$ and $N_{\\rm dof}$ values is to use **Fisher's Gaussian approximation to the chi-squared distribution**: \n",
+    "\n",
+    "$\\;\\;\\;\\;\\;\\sqrt{2\\chi^2_{\\rm min}} \\sim \\mathcal{N}\\left( \\sqrt{2 N_{\\rm dof}-1}, 1 \\right)$\n",
```

Comment:
It's too bad there's no symbol for "approximately tilde".

Reply:
Heh. $\tilde{\sim}$ looks funny but I settled for "(approximately)" after the statement